src/ocm/components/AddHosts/InventoryAddHost.tsx

```diff
@@ -28,7 +28,9 @@ const armArchAlert = (
   />
 );
 
-const InventoryAddHosts: React.FC = () => {
+type InventoryAddHostsProps = { isSNO: boolean };
+
+const InventoryAddHosts: React.FC<InventoryAddHostsProps> = ({ isSNO }: InventoryAddHostsProps) => {
```

Comment:
seem to me it doesn't need to be passed as a property since you have the cluster